R/PqConnection.R

```diff
@@ -205,6 +205,9 @@ setMethod("dbConnect", "PqDriver",
     )
     on.exit(dbDisconnect(conn))
 
+    # set datestyle workaround - https://github.com/r-dbi/RPostgres/issues/287
+    dbExecute(conn, "SET datestyle to iso, mdy;")
```

Comment:
The semicolon feels unnecessary.

```suggestion
    dbExecute(conn, "SET datestyle to iso, mdy")
```

Reply:
removed in latest commit.